src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -43,7 +43,8 @@ setTimeout(async function () {
       const payload = JSON.parse(message.data);
       const action = {
         'UpdateStaticFile': () => updateStaticFile(payload.path),
-        'BlazorHotReloadDeltav1': () => applyBlazorDeltas(payload.sharedSecret, payload.deltas),
+        'BlazorHotReloadDeltav1': () => applyBlazorDeltas(payload.sharedSecret, payload.deltas, false),
+        'BlazorHotReloadDeltav2': () => applyBlazorDeltas(payload.sharedSecret, payload.deltas, true),
```

Comment:
I preserved backward compat here, but I'm not sure what the versioning requirements actually here. Can this script be used across multiple versions of dotnet-watch? If not, why have version in the action name at all?

Reply:
I had a similar thought. I guess I could imagine a scenario where the injection occurs from one version of the SDK and dotnet-watch is run from another (particularly with local copies of the SDK like we have with arcade)? But it seems like you'd have to jump through some hoops to make that happen.